keps/sig-scheduling/3633-matchlabelselectors-to-podaffinity/README.md

```diff
@@ -310,30 +341,89 @@ required) or even code snippets. If there's any ambiguity about HOW your
 proposal will be implemented, this is the place to discuss them.
 -->
 
-A new optional field `MatchLabelKeys` is introduced to `PodAffinityTerm`.
+A new optional field `MatchLabelSelectors` is introduced to `PodAffinityTerm`.
 
 ```go
+type LabelSelectorOperator string
+
+type MatchLabelSelector struct {
+  // Key is used to lookup value from the incoming pod labels, 
+  // and that key-value label is merged with `LabelSelector`.
+  // Key that doesn't exist in the incoming pod labels will be ignored. 
+  Key  string
+  // Operator defines how key-value, fetched via the above `Keys`, is merged into LabelSelector.
+  // If Operator is `In`, `key in (value)` is merged with LabelSelector. 
+  // If Operator is `NotIn`, `key notin (value)` is merged with LabelSelector. 
+  //
+  // +optional
+  Operator       LabelSelectorOperator
+}
+
 type PodAffinityTerm struct {
-	LabelSelector *metav1.LabelSelector
-	Namespaces []string
-	TopologyKey string
-	NamespaceSelector *metav1.LabelSelector
-
-	// MatchLabelKeys is a set of pod label keys to select which pods will
-	// be taken into consideration. The keys are used to lookup values from the
-	// incoming pod labels, those key-value labels are ANDed with `LabelSelector`
-	// to select the group of existing pods which pods will be taken into consideration 
-	// for the incoming pod's pod (anti) affinity. Keys that don't exist in the incoming 
-	// pod labels will be ignored. The default value is empty.
-	// +optional
-	MatchLabelKeys []string
+  LabelSelector *metav1.LabelSelector
+  Namespaces []string
+  TopologyKey string
+  NamespaceSelector *metav1.LabelSelector
+
+  // MatchLabelSelectors is a set of pod label keys to select the group of existing pods 
+  // which pods will be taken into consideration for the incoming pod's pod (anti) affinity. 
+  // The default value is empty.
+  // +optional
+  MatchLabelSelectors []strinMatchLabelSelectorg
```

Comment:
nit: please fix the type

Reply:
Done in https://github.com/kubernetes/enhancements/pull/4069